src/popup_module/renderPopup.js

```diff
@@ -15,23 +17,62 @@ const updateComment = async (id) => {
   });
 };
 
+// Show a message with a type of input
+
+const showMessage = (input, message, type) => {
+  const msg = input.parentNode.querySelector('small');
+  msg.innerText = message;
+  return type;
+};
+
+const showError = (input, message) => {
+  return showMessage(input, message, false);
+};
+
+const showSuccess = (input) => {
+  return showMessage(input, '', true);
+};
+
+//Check input valid
+
+const hasValue = (input, message) => {
```

Comment:
he would be better to use regular expression to validate name input in the form, since now the user can send a comment and has his name with just symbols. 
you could change your function as follows:


`const hasValue = (input, message) => {
  const regExp = /^[\w\s]+$/;
  if (!regExp.test(input.value.trim())) {
    return showError(input, message);
  }
  return showSuccess(input);
};`


<img width="732" alt="Screenshot 2023-02-09 at 07 38 21" src="https://user-images.githubusercontent.com/103900838/217737382-40a11d4d-45e1-4a37-bc4b-79ac62b21b59.png">